src/main/java/usdot/fhwa/stol/c2c/c2c_mvt/controllers/StandardValidationController.java

```diff
@@ -0,0 +1,571 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
```

Comment:
Would Leidos header, refer to: [https://github.com/usdot-fhwa-stol/carma-platform/blob/develop/bsm_generator/src/bsm_generator_node.cpp], apply here

Reply:
Added the header to all java files.